### DISCUSSION TITLE
Stopping button press event fiering continuously in an Entry widget

### DISCUSSION THREAD

**Discussion by Akisailent** [TomSchimansky/CustomTkinter#1360] [General]
I wanted to ask if there was a way to register a keypress event per key press, regardless of how long the key is held down. I know that in tkinter you had the options repeatdelay and repeatinterval to manage this. Can this be also replicated somehow for CTkEntry widget?

**Comment by avalon60** [comment]
I know that there were some event binding limitations in CustomTkiner v4. I think @TomSchimansky filled some gaps in v5, e.g button binding. That said, I'm not sure whether your particular requirement is met. You may need to raise it as an issue.
